app/place/tests.py

```diff
@@ -22,3 +29,23 @@ def test_location_label(self):
     place = Place.objects.get(id=1)
     label = place._meta.get_field('location').verbose_name
     self.assertEqual(label, 'Location')
+  
+  def test_update_screens_button_renders(self):
+    User.objects.create_superuser(test_username, test_email, test_password)
+
+    c = Client()
+    c.login(username=test_username, password=test_password)
+    response = c.get('/admin/place/place/')
+  
+    self.assertEqual(response.status_code, 200)
+    self.assertTrue(update_screens_button_label in response.content.decode('utf-8'))
+
+  def test_click_update_screens_button_result(self):
+    User.objects.create_superuser(test_username, test_email, test_password)
+
+    c = Client()
+    c.login(username=test_username, password=test_password)
+    response = c.get('/admin/place/place/actions/UpdateScreens/', follow=True)
```

Comment:
Need to add `with disable_auto_indexing():` here

Reply:
Also added in f468a61.